src/modules/actions/index.jsx

```diff
@@ -67,13 +67,19 @@ export const hr = () => {
   }
 }
 
-export const trash = ({ t, pushModal, popModal, hasWriteAccess, refresh }) => {
-  const label = t('SelectionBar.trash')
+export const trash = ({
+  t,
+  pushModal,
+  popModal,
+  hasWriteAccess,
+  refresh,
+  byDocId,
+  isOwner
+}) => {
   const icon = TrashIcon
 
   return {
     name: 'trash',
-    label,
```

Comment:
I think you should keep the `label` has this action menu is also used into the SelectionBar

Reply:
The `label` property no longer seems to be used in either `ActionsMenu` or `ActionsBar`.